code/datums/components/defibrillator.dm

```diff
@@ -165,7 +165,7 @@
 	if(ghost?.can_reenter_corpse)
 		to_chat(ghost, "<span class='ghostalert'>Your heart is being defibrillated. Return to your body if you want to be revived!</span> (Verbs -> Ghost -> Re-enter corpse)")
 		window_flash(ghost.client)
-		ghost << sound('sound/effects/genetics.ogg')
```

Comment:
Im pretty sure wrapping in `sound()` removes the reverb that happens at times. This might be intentional.

Reply:
I'll add the wrapper back then, that's pretty odd though

Reply:
wrappers readd